### ISSUE TITLE
$disconnect typing as missing

### ISSUE THREAD

**Issue by izziaraffaele** [prisma/prisma#3270]
## Bug description
I just updated nexus-plugin-prisma which bundles prisma@2.4.1 and I'm getting a warning on the terminal which informs me to switch to prisma.$disconnect() instead of prisma.diconnect(). 

But as soon as I use prisma.$disconnect() typescript complains that prisma.$disconnect type definitions are missing. If you think this is a problem of nexus-plugin-prisma just close this issue and I'll repost it on nexus-plugin-prisma repo.

## Environment & setup

- OS: Mac OS
- Database: PostgreSQL
- Node.js version: v14.4.0
- Prisma version:
```
@prisma/cli          : 2.4.1
Current platform     : darwin
Query Engine         : query-engine 195d4bdc2d16132977f4ba7a8ca312f7906cb086 (at node_modules/@prisma/cli/query-engine-darwin)
Migration Engine     : migration-engine-cli 195d4bdc2d16132977f4ba7a8ca312f7906cb086 (at node_modules/@prisma/cli/migration-engine-darwin)
Introspection Engine : introspection-core 195d4bdc2d16132977f4ba7a8ca312f7906cb086 (at node_modules/@prisma/cli/introspection-engine-darwin)
Format Binary        : prisma-fmt 195d4bdc2d16132977f4ba7a8ca312f7906cb086 (at node_modules/@prisma/cli/prisma-fmt-darwin)
```

**Comment by pantharshit00** [comment]
Yes, this does look like a nexus issue. I wasn't able to reproduce this in a simple script. 
![image](https://user-images.githubusercontent.com/22195362/90009391-9846e700-dcbb-11ea-9672-c6be5c1bf60f.png)

Also, I can find the method in `node_modules/.prisma/index.d.ts`
![image](https://user-images.githubusercontent.com/22195362/90009515-d0e6c080-dcbb-11ea-9073-a2e50c3f4ba2.png)


You can also check that file to verify the typings.
